operations/iuf/examples/iuf_list_stages.md

```diff
@@ -0,0 +1,28 @@
+# `iuf list-stages` Examples
+
+(`ncn-m001#`) List the stages for activity `joe-install-20230107-2` along with their status and duration values.
+
+```bash
+iuf -a joe-install-20230107-2 list-stages
++----------------------------+--------------------------------------------------------------------------------------------+-----------+----------+
+| stage                      | description                                                                                | status    | duration |
++----------------------------+--------------------------------------------------------------------------------------------+-----------+----------+
+| process-media              | Inventory and extract products in the media directory for use in subsequent stages         | Succeeded | 0:01:30  |
+| pre-install-check          | Perform pre-install readyness checks                                                       | Succeeded | 0:00:22  |
+| deliver-product            | Upload product content onto the system                                                     | Succeeded | 0:07:14  |
+| update-vcs-config          | Merge working branches and perform, automated VCS configuration                            | N/A       | N/A      |
+| update-cfs-config          | Update CFS configuration (sat bootprep run --config)                                       | N/A       | N/A      |
+| prepare-images             | Build and configure management node and/or managed node images (sat bootprep run --images) | N/A       | N/A      |
+| management-nodes-rollout   | Rolling reboot or liveupdate of management nodes                                           | N/A       | N/A      |
+| deploy-product             | Deploy services to system                                                                  | N/A       | N/A      |
+| post-install-service-check | Perform post-install checks of processed services                                          | N/A       | N/A      |
+| managed-nodes-rollout      | Rolling reboot or liveupdate of managed nodes nodes                                        | N/A       | N/A      |
```

Comment:
Delete redundant "nodes"

Reply:
Will be addressed when we update to a newer example of `iuf` output.  These typos were in the code itself and have since been fixed.

Reply:
fixed